.vscode/settings.json

```diff
@@ -7,5 +7,7 @@
   "editor.codeActionsOnSave": {
     "source.fixAll.eslint": true
   },
-  "typescript.tsdk": "node_modules/typescript/lib"
+
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true
```

Comment:
This should prompt VSCode users when the version of TS they're using is not the same one as the project's TS version.